### DISCUSSION TITLE
Alt+arrows to move cursor: weird behavior on MacOS+WezTerm

### DISCUSSION THREAD

**Discussion by tgross35** [zellij-org/zellij#3261] [Q&A]
I am using Wezterm with the following configuration:

```lua
config.keys = {
  {
    key = 'LeftArrow',
    mods = 'ALT',
    action = wezterm.action.SendKey {
      key = 'b',
      mods = 'ALT',
    },
  },
  {
    key = 'RightArrow',
    mods = 'ALT',
    action = wezterm.action.SendKey {
      key = 'f',
      mods = 'ALT',
    },
  }
}
```

This allows me to use the arrow keys to move left and right one word at a time, and works as expected when not in Zellij.

However, in Zellij, this seems to break for some reason. Option+leftarrow works fine and moves the cursor to the left, but option+rightarrow changes into something like "jump to next character or delete" depending on what I hit next - pressing the right arrow deletes to the end of the line, entering a character puts the cursor at the next instance of that character.

Does anyone know where this behavior would be coming from or how I could fix it?

**Comment by tgross35** [comment]
Okay, interesting tidbit - it seems like this is specific to zsh, which is the default on MacOs. Launching Bash, Fish, Nushell all work correctly.

I will open an issue since it seems like something is zellij+zsh-specific here. _edit: looks like somebody already did https://github.com/zellij-org/zellij/issues/2908_

**Comment by vietvudanh** [comment]
Exact same problem on WezTerm. I am using fish.
